hummingbot/core/data_type/order_book_tracker_data_source.py

```diff
@@ -220,3 +221,6 @@ async def _sleep(self, delay):
 
     def _time(self):
         return time.time()
+
+    async def _on_order_stream_interruption(self, websocket_assistant):
```

Comment:
Missing type hint (please check also in the other classes)

Reply:
Added type hints